examples/cross_val_fashion.py

```diff
@@ -0,0 +1,77 @@
+"""Simple Convolution and fully connected blocks example."""
```

Comment:
since this is an example script - can you add some more inline comments about what the  various sections do? would be easier to follow

Reply:
Just did!

Reply:
that's a great start! but for each major block of code, we need comments so that you can be able to understand the entire flow of the example from just the inline docs. thanks!